documentation/cobigen-core_configuration.asciidoc

```diff
@@ -354,7 +354,7 @@ configure multiple (comma separated) `groupIds`. By default, (public) CobiGen `g
 template-sets.groupIds=com.devonfw.cobigen.templates,jaxen,jakarta.xml.bind
 template-sets.allow-snapshots=true
 template-sets.disable-default-lookup=false
-template-sets.hide=com.devonfw(:test-artifact(:3.2.1-SNAPSHOT))
+template-sets.hide=com.devonfw.cobigen:core-parent:2021.12.007-SNAPSHOT
```

Comment:
I would use a different example here for the documentation since `com.devonfw.cobigen:core-parent` is a valid expression but makes no sense (cobigen core does not contain template-sets). Use either a hypothetical example or one of the standard CobiGen template sets.

Reply:
com.devonfw.cobigen.templates:templates-parent:2021.12.006-SNAPSHOT
com.devonfw.cobigen.templates:crud-angular-client-app:2021.12.007-SNAPSHOT

These will be valid examples correct ?